tap_wikipedia/models/types/abstract.py

```diff
@@ -0,0 +1,6 @@
+from typing import Annotated
+
+from pydantic import Field
+
+# A tiny type for a Wikipedia article's abstract.
+Abstract = Annotated[str, Field(json_schema_extra={"strip_whitespace": "True"})]
```

Comment:
Nit: In a lot of my projects I've had a tiny type called `NonBlankString` and just re-aliased it everywhere. You could probably do something similar here.